ldap_validator/test/test_data/yaml_data/testsitedefault_keyerror.yml

```diff
@@ -0,0 +1,23 @@
+# Internal SAS site defaults
```

Comment:
This makes me uneasy.   Implies you are about to actually see some SAS internal information.

Reply:
We may need to change and maybe remove these files based on recent feedback from Technical Writing. We can discuss further via the review meeting.

Reply:
Will be removed with upcoming push.